ros/src/sensing/fusion/packages/autoware_camera_lidar_calibrator/CMakeLists.txt

```diff
@@ -46,6 +46,13 @@ include_directories(
 link_directories(${PCL_LIBRARY_DIRS})
 link_directories(${OpenCV_LIBRARY_DIRS})
 
+install(TARGETS
+  autoware_camera_lidar_calibration_node
+  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
+  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION}
+)
+
```

Comment:
The install directive needs to be below where the target (autoware_camera_lidar_calibration_node) is defined. Please move it to just above the `install(DIRECTORY launch/` directive.

Reply:
Yes, my bad. Fixed.